### DISCUSSION TITLE
Access to roles and permissions

### DISCUSSION THREAD

**Discussion by exshapoval** [spatie/laravel-permission#2340] [Q&A]
Hi

How I can setup access to roles and permissions by some role or permission?

**Comment by erikn69** [comment]
I'm not sure if I understood correctly, but maybe this information can help you: [Defining a Super-Admin](https://spatie.be/docs/laravel-permission/v5/basic-usage/super-admin)

**Reply by exshapoval** [reply]
Hi
I mean that I want to give access to roles and permission pages

/admin/resources/roles
/admin/resources/permissions

by some role or permission

**Reply by erikn69** [reply]
Many ways, read documentation
https://spatie.be/docs/laravel-permission/v5/basic-usage/middleware
https://spatie.be/docs/laravel-permission/v5/best-practices/using-policies
https://qcode.in/easy-roles-and-permissions-in-laravel-5-4/

**Reply by exshapoval** [reply]
Not sure that is what I need

I create Policies/RolePolicy.php

setup some to return false to testing and stiil see pages
